test/unit/node_stream_spec.js

```diff
@@ -25,66 +24,28 @@ if (!isNodeJS) {
 }
 
 describe("node_stream", function () {
-  let tempServer = null;
-
   const url = process.getBuiltinModule("url");
   const cwdURL = url.pathToFileURL(process.cwd()) + "/";
   const pdf = new URL("./test/pdfs/tracemonkey.pdf", cwdURL).href;
   const pdfLength = 1016315;
 
-  beforeAll(function () {
-    tempServer = createTemporaryNodeServer();
-  });
-
-  afterAll(function () {
-    // Close the server from accepting new connections after all test finishes.
-    const { server } = tempServer;
-    server.close();
-
-    tempServer = null;
-  });
-
-  it("read both http(s) and filesystem pdf files", async function () {
-    const stream1 = new PDFNodeStream({
-      url: `http://127.0.0.1:${tempServer.port}/tracemonkey.pdf`,
-      rangeChunkSize: 65536,
-      disableStream: true,
-      disableRange: true,
-    });
-
+  it("read filesystem pdf files", async function () {
     const stream2 = new PDFNodeStream({
```

Comment:
The `2` suffix for all variables in the affected tests here might be a bit confusing to future readers because there is no `1` version anymore. Would it be better to remove the suffix in the touched tests? It would also simplify e.g. `range21Reader` to `range1Reader` so the intent is more clear.